Assets/Scripts/CarController.cs

```diff
@@ -39,54 +39,44 @@ public class CarController : MonoBehaviour
     public AudioSource engineSFX, driftingSFX;
     public float driftingFadeRate = 2f;
 
-    //  Lap Record and Checkpoint Record
+    // race management related variables
     private int nextCheckpoint;
     public int currentLap;
-
-
-
-    //  Lap Time Record
     public float lapTime, bestLapTime;
 
 
-
-
-
-
-
-
     //  Start is called before the first frame update
     void Start()
     {
         // separate rigidbody from the car at the start
         rigidBody.transform.parent = null;
         // setting the drag on ground to be equal to that of the sphere
         dragOnGround = rigidBody.drag;
-
-
-
-        //  This sets the start count to 1 and the total lap to the supposed value for the lapDisplay
-        UIManager.instance.LapCounterText.text = currentLap + "/" + RaceManager.instance.totalLaps;
+        // display correct current lap count
+        UIManager.instance.lapCounterText.text = currentLap + "/" + RaceManager.instance.totalLaps;
     }
 
+
     // Update is called once per frame
     void Update()
```

Comment:
Have changed all function called in `Update()` to capital U, since it seems like all methods in C# is preferred to be in PascalCase rather than camelCase.